### ISSUE TITLE
Error when running in watch mode

### ISSUE THREAD

**Issue by nojaf** [jgrund/fable-jest-template#2]
I tried the watch mode as explained in the readme.
Node crashes and I get:

```
Determining test suites to run...Error: This promise must be present when running with -o.
    at C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\search_source.js:202:17
    at next (native)
    at step (C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\search_source.js:19:362)
    at C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\search_source.js:19:592
    at C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\search_source.js:19:273
    at SearchSource.getTestPaths (C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\search_source.js:216:10)
    at C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\run_jest.js:44:29
    at next (native)
    at step (C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\run_jest.js:23:380)
    at C:\Temp\MyTests\node_modules\jest\node_modules\jest-cli\build\run_jest.js:23:540
```

**Comment by jgrund** [comment]
Interesting, looks like it's this issue: https://github.com/facebook/jest/issues/4419

I'll change the flag to `watchall` which should circumvent the issue until the fix lands.

**Comment by jgrund** [comment]
Released 0.0.3 to address.
